common/url.go

```diff
@@ -313,6 +313,26 @@ func (c URL) ServiceKey() string {
 	return buf.String()
 }
 
+func (c *URL) ColonSeparatedKey() string {
+	intf := c.GetParam(constant.INTERFACE_KEY, strings.TrimPrefix(c.Path, "/"))
+	if intf == "" {
+		return ""
+	}
+	buf := &bytes.Buffer{}
+	buf.WriteString(intf)
+	buf.WriteString(":")
+	version := c.GetParam(constant.VERSION_KEY, "")
+	if version != "" && version != "0.0.0" {
+		buf.WriteString(version)
+	}
+	group := c.GetParam(constant.GROUP_KEY, "")
+	buf.WriteString(":")
+	if group != "" {
+		buf.WriteString(group)
+	}
+	return buf.String()
+}
+
 func (c *URL) EncodedServiceKey() string {
```

Comment:
Why not use ServiceKey() func in url.go

Reply:
As far as I know, many places including the latest version of dubbo admin use the `ColonSeparatedKey` method to get the unique  key of the url.

Reply:
I means that ServiceKey method is designed for getting the unique key of the url.

Reply:
Is the method ServiceKey is not satisfy for you?